{
  try {
    action.call();
  }
 catch (  Exception ignored) {
  }
}
